tests/Model/BasePHPElement.php

```diff
@@ -52,35 +58,34 @@ protected function getFQN(Node $node): string
         return rtrim($fqn, "\\");
     }
 
-    protected static function convertReflectionTypeToString(?ReflectionType $type): string
+    protected static function getReflectionTypeAsArray(?ReflectionType $type): array
     {
+        $reflectionTypes = [];
         if ($type instanceof ReflectionNamedType) {
-            return (string)$type;
+            $type->allowsNull() && $type->getName() !== 'mixed' ?
```

Comment:
Doesn't this work `array_push($reflectionTypes, (string) $type)`?

Reply:
Expected this question :) `__toString` is deprecated 
```
#[Deprecated(replacement: "%class$->getName()", since: "7.1")]
    public function __toString()
    {
    }
```
that's why decided to get rid of it now, to avoid future problems